plugin/src/main/java/org/kravemir/gradle/sass/AbstractSassCompileTask.java

```diff
@@ -36,7 +38,7 @@ FileCollection getSassFiles() {
         return fileTree;
     }
 
-    @InputFiles
+    @Internal
```

Comment:
I don't think `getInputFiles` is necessary anymore. As whole `getSrcDir` is now `@InputDirectory`, then this method is not needed anymore.

It was there previously to make task rebuild CSS, if some `@import ...` file changes.